RELEASE_NOTES.md

```diff
@@ -26,7 +26,8 @@ In this version we are changing Decidim's underlaying configuration engine, so,
 ```bash
 git rm config/secrets.yml
 git rm config/initializers/decidim.rb
-wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml.erb -O config/storage.yml
+wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml -O config/storage.yml
+```
```

Comment:
```suggestion
Also, the `config/storage.yml` needs to be updated to remove any references to the `Decidim.application.secrets` object. If you have never modified this file you can safely execute the following command to update the file (**DON'T DO IT IF YOU HAVE CUSTOMIZED IT**, check section 3.4 in that case):

```bash
wget https://raw.githubusercontent.com/decidim/decidim/refs/heads/develop/decidim-generators/lib/decidim/generators/app_templates/storage.yml -O config/storage.yml
```
See section 
```

Reply:
I don't want to start **SHOUTING AND SCREAMING IN THE RELEASE NOTES**, so I'll change that format. 

Mind that any of the instructions that we give in for the update process are important and can potentially break the app 😄

Reply:
Also we have the same problem (customization by implementers) for the other files, so I'm adding a general disclaimer. Check out 05c46c8416 please and let me know if that works for you!